src/Compilers/Core/Portable/InternalUtilities/CompilerOptionParseUtilities.cs

```diff
@@ -11,14 +13,14 @@ internal static class CompilerOptionParseUtilities
         /// Parse the value provided to an MSBuild Feature option into a list of entries.  This will 
         /// leave name=value in their raw form.
         /// </summary>
-        public static IList<string> ParseFeatureFromMSBuild(string features)
+        public static IList<string> ParseFeatureFromMSBuild(string? features)
         {
             if (string.IsNullOrEmpty(features))
```

Comment:
Consider using the new RoslynString helper to remove the suppression.

Reply:
➡️ Fixed